SmartCar/Arduino/sketch_main/sketch_main.ino

```diff
@@ -57,6 +68,36 @@ void setup() {
     pinMode(FLAME_SENSOR_PIN, INPUT);
     
     Serial.begin(9600);  
+
+    // Start the sensor
+    sensors.begin();
+    
+    if (!sensors.getAddress(thermometer, 0)) Serial.println("Unable to find address for Device 0"); 
+  
+  // set the resolution to 9 bit
+    sensors.setResolution(thermometer, 9);
+    sensors.getResolution(thermometer);
```

Comment:
Is the get really necessary? you are setting the resolution on the line above and you are not using the value for anything